spread.yaml

```diff
@@ -55,6 +55,29 @@ backends:
                 username: ubuntu
                 password: ubuntu
 
+    adhoc-ubuntu-core:
+        type: adhoc
+        environment:
+            SPREAD_ADHOC_UC_ADDRESS: "$(HOST: echo ${SPREAD_ADHOC_UC_ADDRESS:-localhost})"
+            SPREAD_ADHOC_UC_PORT: "$(HOST: echo ${SPREAD_ADHOC_UC_PORT:-8022})"
+        allocate: |
+            echo "Allocating $SPREAD_SYSTEM..."
+            ADDRESS $SPREAD_ADHOC_UC_ADDRESS:$SPREAD_ADHOC_UC_PORT
+        discard: |
+            echo "Discarding ad-hoc $SPREAD_SYSTEM"
+        restore: |
+            rm -rf $GOPATH/src
```

Comment:
Without this, when i try to run the suite again on the same instance after an execution i get:
```
2016/09/20 10:21:19 Sending project data to external-ubuntu-core:ubuntu-core-16-device-amd64...
2016/09/20 10:21:19 Discarding external-ubuntu-core:ubuntu-core-16-device-amd64, cannot send project data: remote directory /home/gopath/src/github.com/snapcore/snapd is not empty
```
If I try `-keep` I get a password that doesn't work (even if I use the password `ubuntu` declared in the backend it seems to try to connect with the root user), and i can not include `-reuse` without password.